README.md

```diff
@@ -12,6 +12,13 @@ upstreaming.
 It moves the P4-specific components of the architecture to a separate
 process called `infrap4d`.
 
+## Upcoming changes
+
+The Stratum dependencies will be moving from the `setup` directory to their
```

Comment:
typo: their 2 times?

Reply:
Good catch. Fixed. Made a couple of clarifying edits at the same time.